fastapi/openapi/utils.py

```diff
@@ -393,6 +393,7 @@ def get_openapi(
     terms_of_service: Optional[str] = None,
     contact: Optional[Dict[str, Union[str, Any]]] = None,
     license_info: Optional[Dict[str, Union[str, Any]]] = None,
+    prefix: Optional[str] = None,
```

Comment:
Why not empty string as default value? That will make if statement redundant.

Reply:
Thanks!